test/e2e/data/ironic-deployment/overlays/pr-test/kustomization.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: baremetal-operator-system
+namePrefix: baremetal-operator-
+resources:
+- https://github.com/metal3-io/baremetal-operator/config/namespace?ref=release-0.10
+- https://github.com/metal3-io/baremetal-operator/ironic-deployment/base?ref=release-0.10
+
+components:
+- https://github.com/metal3-io/baremetal-operator/ironic-deployment/components/basic-auth?ref=release-0.10
+- https://github.com/metal3-io/baremetal-operator/ironic-deployment/components/tls?ref=release-0.10
+
+configMapGenerator:
+- behavior: create
+  envs:
+  - ironic_bmo_configmap.env
+  name: ironic-bmo-configmap
+
+patches:
+- path: keepalived_patch.yaml
+
+images:
+- name: quay.io/metal3-io/ironic
+  newTag: release-29.0
```

Comment:
Same here, 29.0?

Reply:
The default for release-1.10 is 29.0. 
Both name and tag will be replaced by the image that was used by metal3-dev-env.

Reply:
Sure, but why not default to main+main?

Reply:
Because this PR is on capm3 release-1.10, it's better to use the defaults that are for release-1.10 capm3.
On the main branch, we can use main+main.